recipes-extended/xen/files/libxl-cleanup-string-to-backend.patch

```diff
@@ -0,0 +1,62 @@
+################################################################################
+SHORT DESCRIPTION:
+################################################################################
+Remove deprecated and unreachable sequence from libxl_string_to_backend.
+
```

Comment:
This patch can be removed, it was part of the series on libxl that was PR'ed separately (https://github.com/OpenXT/xenclient-oe/pull/1092).
It was determined this should be submitted upstream eventually.

Reply:
Removed.